jaworski_cellseg/label_counting_widget.py

```diff
@@ -18,23 +62,35 @@ def create_label_counting_widget(viewer: napari.Viewer):
             "label": "Total count",
             "enabled": False,
         },  # Make the textbox read-only
+        save_results={
+            "widget_type": "FileEdit",
+            "mode": "w",
+        },  # Dynamic choices will be set later
     )
     def cell_counting_widget(
         label_layer_name: str = None,
```

Comment:
If a parameter can be `None`, then the type is either `Optional[str]` or `str | None` (after Python 3.10)

Reply:
so are we only supporting python >3.10 ?